### ISSUE TITLE
Generated Doxygen Comment randomly failed "Actual command not found, wanted to execute C_Cpp.GenerateDoxygenComment/123"

### ISSUE THREAD

**Issue by sean-mcmanus** [microsoft/vscode-cpptools#10217]
![image](https://user-images.githubusercontent.com/19859882/205184177-5b500a7f-4b5e-4cf5-bd95-4a57708b2951.png)

Repro was with 1.13.6 (unreleased yet). I don't know the repro details yet. It seemed random. It could be some VS Code bug (I used Insiders).

**Comment by sean-mcmanus** [comment]
I still repro this randomly, but I don't know how to get it to repro again (to debug it).

<img width="904" height="166" alt="Image" src="https://github.com/user-attachments/assets/5195248c-2243-40d0-ab49-3423ea899539" />

**Comment by bobbrow** [comment]
Is the extension activated?

**Comment by sean-mcmanus** [comment]
@bobbrow Yes, the extension has to be activated in order to generate the code lens with the Generate Doxygen Comment code lens in it.
